maestro-orchestra-models/src/main/java/maestro/orchestra/util/Env.kt

```diff
@@ -8,7 +8,7 @@ import maestro.orchestra.MaestroCommand
 object Env {
 
     fun String.evaluateScripts(jsEngine: JsEngine): String {
-        val result = "(?<!\\\\)\\\$\\{([^\$]*)}".toRegex()
```

Comment:
I wonder what the original author intended with the original regex here?

Reply:
Yes, I believe it was to detect anything that's encosed in `${}` and that does not contain `$` inside

Reply:
But why would they have done specific work to exclude the dollar, if it were as simple as just `.*?` ? 
Is there an additional risk we've not considered?